common/src/theme/BlogPostItem/Header/Title/styles.module.css

```
@@ -2,6 +2,16 @@
   font-size: 3rem;
 }
 
+.title.postTitle {
```

Stylelint (17.14.0)

[error] Expected class selector ".postTitle" to be kebab-case (selector-class-pattern)

(selector-class-pattern)